src/main/java/org/sonarlint/intellij/ui/SonarLintRulePanel.java

```diff
@@ -26,6 +26,9 @@
 import com.intellij.openapi.util.text.StringUtil;
 import com.intellij.ui.IdeBorderFactory;
 import com.intellij.ui.SideBorder;
+import com.intellij.util.IconUtil;
+import com.intellij.util.ui.JBUI;
+import com.intellij.util.ui.UIUtil;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.intellij.util.ui.UIUtil'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)